packages/core/lib/modules/transactions.test.js

```diff
@@ -0,0 +1,9 @@
+// import { Transactions } from './transactions';
+
+describe('TransactionsModule', () => {
```

Comment:
this is another one where I want to see high coverage

Reply:
Sure https://www.notion.so/dockteam/Wallet-sdk-code-review-25d2f2d7644c470ea5a1012a8cd7c871